airbyte-proxy/nginx-auth.conf.template

```diff
@@ -42,4 +42,26 @@ http {
       }
     }
   }
+
+#server {
+#    listen 8003;
+
+#    location / {
+#      proxy_set_header Host $host;
+#      proxy_set_header X-Real-IP $remote_addr;
+#      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+#      auth_basic "Welcome to Airbyte";
+#      auth_basic_user_file /etc/nginx/.htpasswd;
+
+#      proxy_pass "${CONNECTOR_BUILDER_SERVER_API}";
```

Comment:
forward to "[http://airbyte-connector-builder-server:80](http://airbyte-connector-builder-server/)" as defined in https://github.com/airbytehq/airbyte/pull/19264/files#diff-8a4136717271f90d70120fef7bb8904d26f3fbcfe7520dbd39fa0b8723d8548eR15